swap/tests/happy_path_restart_bob.rs

```diff
@@ -32,3 +32,31 @@ async fn given_bob_restarts_after_xmr_is_locked_resume_swap() {
     })
     .await;
 }
+
+#[tokio::test]
+async fn given_bob_restarts_before_xmr_is_locked_resume_swap() {
```

Comment:
In the past the test execution could not handle tests in the same file. Would be great if that works now.

Reply:
It doesn't :(